diff --git a/node_modules/turbowatch/dist/backends/ChokidarWatcher.js b/node_modules/turbowatch/dist/backends/ChokidarWatcher.js
index e6e49ea..a4fab4c 100644
--- a/node_modules/turbowatch/dist/backends/ChokidarWatcher.js
+++ b/node_modules/turbowatch/dist/backends/ChokidarWatcher.js
@@ -38,6 +38,7 @@ class ChokidarWatcher extends FileWatchingBackend_1.FileWatchingBackend {
             log.trace('indexed %s %s...', discoveredFileCount.toLocaleString('en-US'), discoveredFileCount === 1 ? 'file' : 'files');
         }, 5000);
         this.chokidar = chokidar.watch(project, {
+            ignored: ['**/node_modules/**'],
             awaitWriteFinish: false,
             followSymlinks: true,
         });
